### PR TITLE
[Serverless] Updates D4C badge from tech preview to beta 

### LINKED ISSUE #5454: [BUG] Update D4C overview intro / tag
### Documentation links

This doc https://www.elastic.co/guide/en/security/current/d4c-overview.html should specify that D4C is in beta. Affects both serverless and ESS

### Description

Missing beta tag

### Which documentation set(s) does this bug apply to?

ESS and serverless

### Release version

Goes back to 8.8

### Testing environment

-

### PR DESCRIPTION
**Pull request by benironside**
Fixes #5454 by updating the tech preview badge to beta for D4C docs (serverless)

Previews: [D4C overview](https://elastic-dot-co-docs-production-3icqoy3dk-elastic-dev.vercel.app/current/serverless/security/d4c-overview), [get started with D4C](https://elastic-dot-co-docs-production-3icqoy3dk-elastic-dev.vercel.app/current/serverless/security/d4c-get-started)

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-6fxu9d7r4-elastic-dev.vercel.app

* Built with commit 3a3eb7b832a69135650cebca3c717db84ec8195f

> Issues? Visit #next-docs in Slack

File: docs/serverless/cloud-native-security/d4c-get-started.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ tags: ["security","how-to","get-started", "cloud security"]
 status: in review
 ---
 
-<DocBadge template="technical preview" />
+<DocBadge template="beta" />
```
Comment (joepeeples) [inline docs/serverless/cloud-native-security/d4c-get-started.mdx:9]:
```suggestion
<DocBadge template="technical preview" />

<DocCallOut template="beta" />
```

File: docs/serverless/cloud-native-security/d4c-overview.mdx [hunk 2]
```diff
@@ -6,7 +6,7 @@ tags: ["security","cloud","reference","manage"]
 status: in review
 ---
 
-<DocBadge template="technical preview" />
+<DocBadge template="beta" />
```
Comment (joepeeples) [inline docs/serverless/cloud-native-security/d4c-overview.mdx:9]:
```suggestion
<DocBadge template="technical preview" />

<DocCallOut template="beta" />
```

**Review by joepeeples — COMMENTED** [review]
Soooo... the DocBadge at the top of every serverless docs page should remain as technical preview, because all of serverless is still considered technical preview. 

We never really worked out a great way to handle release states for individual features within serverless, but one option is to add a callout that identifies the feature's state. I think the syntax I suggested should do this, but might want to test it first.

**Review by joepeeples — APPROVED** [review]

**Pull request merged**